snuba/clickhouse/translators/snuba/mappers.py

```diff
@@ -134,6 +134,25 @@ def __init__(
         )
 
 
+@dataclass(frozen=True)
+class ColumnToUserNullIf(ColumnToFunction):
+    """
+    Custom column mapper for mapping user columns to null.
+    TODO: Can remove when we support dynamic expression parsing in config
+    """
+
+    def __init__(
+        self, from_table_name: str, from_col_name: str, to_function_name: str
+    ) -> None:
+        to_function_params: Tuple[ColumnExpr, LiteralExpr] = (
+            ColumnExpr(None, None, "user"),
+            LiteralExpr(None, ""),
+        )
+        super().__init__(
+            from_table_name, from_col_name, to_function_name, to_function_params
+        )
```

Comment:
Could this be more generic like `ColumnToNullIf`? I'm guessing there may be other instances of `nullIf` function being used aside from just `user` columns.